x-pack/plugins/security_solution/public/management/pages/policy/view/policy_details.test.tsx

```diff
@@ -172,7 +172,7 @@ describe('Policy Details', () => {
       cancelbutton.simulate('click', { button: 0 });
       const navigateToAppMockedCalls = coreStart.application.navigateToApp.mock.calls;
       expect(navigateToAppMockedCalls[navigateToAppMockedCalls.length - 1]).toEqual([
-        'securitySolution:management',
+        'securitySolution:administration',
```

Comment:
:-)
Here as well (maybe global search and replace under `security_solution/public` dir